kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/repository/ConfigFileWatcher.java

```diff
@@ -55,6 +56,8 @@ public ConfigFileWatcher(String configFilePath, ConfigurationMarshaller marshall
             if(toWatch.toFile().exists()) {
                 lastUpdate = Files.getLastModifiedTime(toWatch).toMillis();
             } else {
+                File file = new File(this.toWatch.toString() + "kie-server-router.json");
+                file.createNewFile();
```

Comment:
maybe remove this two lines and use only `toWatch.toFile().createNewFile();` instead?

Reply:
plus, can you add a test to make sure the file is created upon object instantiation?

Reply:
I would create the new file with the configuration provided in the constructor. That would put in sync in memory configuration and file system. 
I would also set the lastUpdate as the new file has.

Reply:
tests?